src/main/java/edu/cornell/kfs/coa/document/validation/impl/AccountReversionGlobalRule.java

```diff
@@ -159,17 +162,14 @@ public boolean processCustomAddCollectionLineBusinessRules(MaintenanceDocument d
         }
         else if (line instanceof AccountReversionGlobalAccount) {
             AccountReversionGlobalAccount acct = (AccountReversionGlobalAccount) line;
-            if (!checkEmptyValue(acct.getChartOfAccountsCode())) {
-                GlobalVariables.getMessageMap().putError("chartOfAccountsCode", KFSKeyConstants.ERROR_REQUIRED, "Chart of Accounts Code");
-            }
-            if (!checkEmptyValue(acct.getAccountNumber())) {
-                GlobalVariables.getMessageMap().putError("accountNumber", KFSKeyConstants.ERROR_REQUIRED, "Account Number");
+            if (!checkEmptyValue(acct.getChartOfAccountsCode()) || !checkEmptyValue(acct.getAccountNumber())) {
+                // Skip Most validation if chart or account are empty. The default required-field checking will populate the error map accordingly.
+                success = false;
```

Comment:
Both the chart code and account number are already configured as required fields, so the other business rules processing will already add appropriate error messages to the screen. This check is just needed to help skip some other validation that should be ignored if they are not both specified.